src/parser/demonhunter/havoc/modules/talents/BlindFury.js

```diff
@@ -70,13 +70,16 @@ class BlindFury extends Analyzer{
       <TalentStatisticBox
         talent={SPELLS.BLIND_FURY_TALENT.id}
         position={STATISTIC_ORDER.OPTIONAL(6)}
-        value={`${this.furyPerMin} fury per min`}
+        value={
+          <>
+            {this.badCast} bad <SpellLink id={SPELLS.EYE_BEAM.id} /> casts<br />
+            {this.furyPerMin} Fury per min
+          </>}
         tooltip={(
           <>
-            Since this will always max out your fury on cast, wasted and totals do not matter. Only the amount effectively gained. <br />
-            A bad cast is when you cast Eye Beam with more than 50 fury. At that point you are wasting enough fury gained for it to be a dps loss. <br /><br />
-            {this.gained} Effective fury gained<br />
-            {this.badCast} Bad casts
+            Since this will always max out your Fury on cast, wasted and totals do not matter. Only the amount effectively gained. <br />
+            A bad cast is when you cast Eye Beam with more than 50 Fury. At that point you are wasting enough fury gained for it to be a dps loss. <br /><br />
```

Comment:
Might as well capitalize DPS here as well since you did elsewhere

Reply:
yup found the other places and capitalized them